app-editors/emacs/emacs-28.1.91-r1.ebuild

```
@@ -317,6 +319,56 @@ src_compile() {
 	emake
 }
 
+src_test() {
+	# List .el test files with a comment above listing the exact
+	# subtests which caused failure. Elements should begin with a %.
+	# e.g. %lisp/gnus/mml-sec-tests.el.
+	local exclude_tests=(
+		# Reason: not yet known
+		# mml-secure-en-decrypt-{1,2,3,4}
+		# mml-secure-find-usable-keys-{1,2}
+		# mml-secure-key-checks
+		# mml-secure-select-preferred-keys-4
+		# mml-secure-sign-verify-1
+		%lisp/gnus/mml-sec-tests.el
+
+		# Reason: race condition
+		# Looks like it should be fixed in 29.x at least:
+		# https://debbugs.gnu.org/cgi/bugreport.cgi?bug=55706
+		# files-tests-file-name-non-special-file-in-directory-p
+		%lisp/files-tests.el
+
+		# Reason: permission denied on /nonexistent
+		# (vz-*-bzr only fails if breezy is installed, as they
```

Typo (twice again below):
```suggestion
		# (vc-*-bzr only fails if breezy is installed, as they
```

---

I saw the same typo in each file - am I missing something else?